src/pages/index.js

```diff
@@ -153,7 +153,7 @@ function handleAvatarChange(data) {
       avatarPopup.renderLoading(false)
       userInfo.setUserInfo(data)
     })
-    .then(avatarPopup.close())
+    .then(avatarPopup.close)
     .finally(avatarPopup.renderLoading(true))
```

Comment:
` .finally(avatarPopup.renderLoading(true))`

This is not correct. See my remark https://github.com/kn8-codes/se_project_aroundtheus/pull/1/files/31becbe36b838625f6d8b7411bc5c3eaae35b01e..686a708f67887fd3f368a4618a0c2e9d90731066#r1354377598 

This is the way it should work (with `false`) - you stop the loading in `finally` rather than start it

```js

    .finally(() => avatarPopup.renderLoading(false))

```